assets/js/googlesitekit/modules/datastore/settings.test.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * `googlesitekit/modules` datastore: settings tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Modules from 'googlesitekit-modules';
+import { STORE_NAME } from './constants';
+import { createTestRegistry } from '../../../../../tests/js/utils';
+
+describe( 'core/modules store changes', () => {
+	let registry;
+	const slug = 'test-module';
+	const nonExistentModuleSlug = 'not-module';
+	const moduleStoreName = `modules/${ slug }`;
+	let moduleCanSubmitChanges = false;
+	let submittingChanges = false;
+
+	beforeAll( () => {
+	} );
+
+	beforeEach( () => {
+		const storeDefinition = Modules.createModuleStore( moduleStoreName );
+		const submitChanges = () => {
+			return {
+				payload: {},
+				type: 'DUMMY_ACTION',
+			};
+		};
+		const canSubmitChanges = () => moduleCanSubmitChanges;
+		const isDoingSubmitChanges = () => {
+			return submittingChanges;
+		};
+
+		registry = createTestRegistry();
+		registry.registerStore( moduleStoreName, Data.combineStores(
+			storeDefinition,
+			{
+				actions: {
+					submitChanges,
+				},
+				selectors: {
+					canSubmitChanges,
+					isDoingSubmitChanges,
+				},
+			}
+		) );
+	} );
+
+	afterAll( () => {
+	} );
+
+	afterEach( () => {
+	} );
+
+	describe( 'selectors', () => {
+		describe( 'submitting changes', () => {
```

Comment:
The describe blocks at this level should mirror the selector name, and then there would be an adjacent `describe` for each selector.
```suggestion
		describe( 'isDoingSubmitChanges', () => {
```

Reply:
Addressed.